### PR TITLE
Adding support for default in ES Modules

### PR DESCRIPTION
**Pull request by megoth**
When adding views it will check for the default property.

This removes the need for `.source.ts`-postfix pattern that we've used when migrating TS-files earlier.

### CODE REVIEW

**Review by Vinnl — APPROVED** [review]
This is vastly preferable to me over the previous workaround. :+1:

**Comment by timbl** [comment]
So we are changing the API for a all panes to a default export?

**Comment by Vinnl** [comment]
@timbl It's an additional API, so panes that use a CommonJS export continue to work, but panes that use regular Javascript exports (and use the default export) would work after this PR as well.

File: src/internal/internalPane.ts [hunk 1]
```diff
@@ -1 +1,255 @@
...
+          noun,
+          function () {
+            if (
+              !confirm(
+                `Are you sure you want to delete ${subject}? This cannot be undone.`
+              )
+            ) {
+              return
+            }
+            // @@ TODO Remove casing of store.fetcher
```
Comment (timbl) [inline src/internal/internalPane.ts:113]: To remove  a casting of store.fetcher and store.updater, how about making superclasses of Store, like ConnectedStore or OnlineStore for the one which has a fetcher, and LiveStore superclass of OnlineStore which would also have an updater ?

Comment (timbl) [reply src/internal/internalPane.ts:113]: Or is it too valuable to be able to make store and attach its helper objects later?

**Review by timbl — APPROVED** [review]

File: src/internal/internalPane.ts (hunk 1 above)
Comment (megoth) [reply src/internal/internalPane.ts:113]: Was thinking the same thing, and have a local commit that's about to be a PR for that. Can push it tomorrow.

Comment (megoth) [reply src/internal/internalPane.ts:113]: Have pushed a PR to `pane-registry` with proposed changes - https://github.com/solid/pane-registry/pull/5

**Pull request merged**